test/integration/scheduler_perf/util.go

```diff
@@ -44,7 +44,7 @@ import (
 // Notes on rate limiter:
 //   - client rate limit is set to 5000.
 func mustSetupScheduler() (schedulerConfigFactory *factory.ConfigFactory, destroyFunc func()) {
-	framework.DeleteAllEtcdKeys()
+	// framework.DeleteAllEtcdKeys()
```

Comment:
can you at least add a comment explaining why it's commented out / under what conditions it should get uncommented? otherwise 3 months from now we're not going to understand why it's there.